.github/workflows/test-integration-amd64.yml

```diff
@@ -106,3 +106,8 @@ jobs:
         if: ${{ always() }}
         continue-on-error: true
         run: GOMODCACHE=$(go env GOMODCACHE) make GO=`which go` test/integration/java
+
+      - name: Integration Tests (Nodejs)
+        if: ${{ always() }}
+        continue-on-error: true
```

Comment:
This shouldn't be the case for NodeJS. We did this for Java because it hasn't been implemented fully yet.